tc-string-parse.js

```diff
@@ -79,6 +79,28 @@
         return result;
     };
 
+    var applySchemaValue = function (schema, value, result) {
+        if (schema.key) {
+            if (schema.parent) {
+                if (!result[schema.parent]) {
+                    result[schema.parent] = {};
+                }
+
+                if (typeof result[schema.parent][schema.key] === "object" && typeof value === "object") {
+                    result[schema.parent][schema.key] = Object.assign({}, result[schema.parent][schema.key], value);
```

Comment:
`Object.assign` is not supported in IE. Could you please extract assign logic from 252-253 lines into additional function and use that?